website2/README.md

```
@@ -1,6 +1,6 @@
 This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
 
-## Getting Started
+## Getting Started .
```

Markdownlint

Punctuation: ' .'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)